mobile/src/main/res/values-de/strings.xml

```diff
@@ -74,6 +74,7 @@
     <string name="action_settings">Einstellungen</string>
     <string name="nfc_dialog_title">Aktion des NFC-Tags</string>
     <string name="info_not_set">Nicht gesetzt</string>
+    <string name="info_settings_moved">Der Menüeintrag Einstellungen wurde in die linke Navigationsleiste verschoben. Bitte nutze diesen Eintrag zukünftig.</string>
```

Comment:
If we aren't going to remove it and keep the message for now:

`Das Einstellungsmenü hat einen neuen Platz: Es kann jetzt über die Navigationsleiste auf der linken Seite erreicht werden.`

Reply:
Removed :)